redaxo/src/core/lib/console/user/delete.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @internal
+ */
+class rex_command_user_delete extends rex_console_command
+{
+    #[Override]
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('login', InputArgument::REQUIRED, 'Login')
+            ->setDescription('Deletes an administrator user by the specified login name.');
```

Comment:
"administrator user" klingt so, als könnte man nur User mit Admin-Flag löschen, aber der Befehl kann ja beliebige backend-user löschen.
Daher würde ich den Begriff "administrator" entfernen (entsprechend auch unten in `deleteAdminUserByGivenLoginName` etc.